src/frameworks/remix.json

```diff
@@ -3,12 +3,14 @@
   "name": "Remix",
   "category": "static_site_generator",
   "detect": {
-    "npmDependencies": ["remix", "@remix-run/netlify"],
+    "npmDependencies": ["remix", "@remix-run/netlify", "@remix-run/netlify-edge"],
```

Comment:
`"@remix-run/netlify-edge"` does not exist yet, but it will soon.

Reply:
Actually, quick question about `npmDependencies`. It means it needs to include all of them correct? So since we'll have a netlify-edge dep soon, should I create an remix-edge.json or just have `npmDependencies` contain only `remix`?

Reply:
OK, so in the ReadME, I confirmed that only one of them needs to be present to be considered the framework. https://github.com/netlify/framework-info#npmdependencies

```
npmDependencies
Type: string[]

Framework's npm packages. Any project with one of those packages in their package.json (dependencies or devDependencies) will be considered as using the framework.

If empty, this is ignored.
```